animation_nodes/nodes/object/object_color_output.py

```diff
@@ -0,0 +1,26 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+
+class ObjectColorOutputNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_ObjectColorOutputNode"
+    bl_label = "Object Color Output"
+    codeEffects = [VectorizedSocket.CodeEffect]
+
+    useObjectList: VectorizedSocket.newProperty()
+    useColorList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput(VectorizedSocket("Object", "useObjectList",
+            ("Object", "object", dict(defaultDrawType = "PROPERTY_ONLY")),
+            ("Objects", "objects"),
+            codeProperties = dict(allowListExtension = False)))
+
+        self.newInput(VectorizedSocket("Color", "useColorList",
+            ("Color", "color"), ("Colors", "colors")))
+
+        self.newOutput(VectorizedSocket("Object", "useObjectList",
+            ("Object", "object", dict(defaultDrawType = "PROPERTY_ONLY")),
```

Comment:
Is `dict(defaultDrawType = "PROPERTY_ONLY")` really needed here?

Reply:
Have you seen this?

Reply:
It is in the _Object Attribute Output_ node.